src/software/test_util/test_util.h

```diff
@@ -17,6 +18,25 @@
 #include "software/world/team.h"
 #include "software/world/world.h"
 
+#define TEST_UTIL_CREATE_MOTION_CONTROL_NO_DEST                                          \
+    [](const Robot &, const Point &) {                                                   \
+        TbotsProto::MotionControl motion_control;                                        \
+        *(motion_control.mutable_requested_destination()) = *createPointProto(Point());  \
+        return motion_control;                                                           \
+    }
+
+#define TEST_UTIL_CREATE_MOTION_CONTROL_WITH_DEST(DESTINATION)                           \
+    [](const Robot &, const Point &) {                                                   \
+        TbotsProto::MotionControl motion_control;                                        \
+        TbotsProto::Path path_proto;                                                     \
+        *(path_proto.add_point())        = *createPointProto(DESTINATION);               \
+        *(path_proto.add_point())        = *createPointProto(DESTINATION);               \
+        *(motion_control.mutable_path()) = path_proto;                                   \
```

Comment:
why do we add two destination points?

could we have robot initial position and then destination point?

Reply:
deleted since unused